src/ButtonSlider.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import leftArrow from './images/left_arrow.svg';
+import rightArrow from './images/right_arrow.svg';
+
+import styled from 'styled-components';
+
+
+const ButtonSliderComponent = ({className, moveSlide, direction}) => {
+    return (
+        <button
+        onClick={moveSlide}
+        className={className}>
+            <img src={direction === "next" ? rightArrow : leftArrow} alt="rightArrow" style={{width:"25px", height:"25px", pointerEvents:"none"}} />
+        </button>
+    )
+};
+
+
+const ButtonSlider = styled(ButtonSliderComponent)`
```

Comment:
This styled component also is better to move into a separate file, it can be something like `ButtonSlider.style.js` and your component be `ButtonSlider.component.jsx`